sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.createOffering.e2e-spec.ts

```diff
@@ -197,6 +198,67 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-createOffering", (
     );
   });
 
+  it("Should throw error when education program is expired.", async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    const fakeEducationProgram = createFakeEducationProgram(
+      {
+        institution: collegeF,
+        user: collegeFUser,
+      },
+      {
+        initialValue: {
+          effectiveEndDate: getISODateOnlyString(new Date()),
+        },
+      },
+    );
+    fakeEducationProgram.sabcCode = faker.random.alpha({ count: 4 });
```

Comment:
Unless required, we should avoid setting the `sabcCode`.
Same for the other E2E tests.

Reply:
removed!